habitat-lab/habitat/datasets/rearrange/modify_episodes_for_object_rearrange.py

```diff
@@ -366,6 +584,8 @@ def add_cat_fields_to_episodes(
         episode["candidate_goal_receps"] = get_candidate_receptacles(
```

Comment:
Why are we not populating the "view_points" list for each of the goals? That way we can directly use ObjectNav's `DistanceToGoal` measure for our task which supports use of viewpoints. I am doing this for Pick reward [here](https://github.com/facebookresearch/habitat-lab/blob/f2e7982a502ee6737c04878f6907656b79a7e9f4/habitat-lab/habitat/tasks/rearrange/sub_tasks/pick_sensors.py#L22).

Reply:
That's what I'm doing [here](https://github.com/facebookresearch/habitat-lab/pull/1112/files#diff-aed663b40b1037a3932fbca619c2911cd788fcfef320f00835e5533bf8b9371dR587-R588).

Reply:
Yeah that should work. Thanks